src/views/CountryRiskWatch/HistoricalDataChart/styles.module.css

```diff
@@ -0,0 +1,90 @@
+.historical-data-chart {
+    .filters {
+        display: grid;
+        grid-gap: var(--go-ui-spacing-xl);
+        grid-template-columns: repeat(auto-fit, minmax(20rem, 1fr));
+    }
+
+    .chart-container {
+        position: relative;
+        background-color: var(--go-ui-color-background);
+        height: 20rem;
+
+        .svg {
+            width: 100%;
+            height: 100%;
+
+            .y-axis-tick-text {
+                text-anchor: start;
+                font-size: var(--go-ui-font-size-xs);
+                dominant-baseline: middle;
+            }
+
+            .x-axis-tick-text {
+                text-anchor: start;
+                transition: var(--go-ui-duration-transition-medium) ease-in-out transform;
+                font-size: var(--go-ui-font-size-xs);
+
+                @media screen and (max-width: 30rem) {
+                    transform: rotate(-30deg);
+                }
+            }
+
+            .x-axis-grid-line,
+            .y-axis-grid-line {
+                stroke: var(--go-ui-color-gray-60);
+                stroke-dasharray: 4 4;
+                stroke-opacity: 0.5;
+
+                &.hovered {
+                    stroke-opacity: 1;
+                }
+            }
+        }
+
+        .point {
+            position: absolute;
+            transform: translate(-50%, -50%);
+            opacity: 0.8;
+            border-radius: 50%;
+            width: 1.4rem;
+            height: 1.4rem;
+
+            .hazard-icon {
+                color: var(--go-ui-color-white);
+                font-size: 1.2rem;
+            }
+
+            &:hover {
+                opacity: 1;
+            }
+        }
+    }
+
+    .legend {
+        display: flex;
+        gap: var(--go-ui-spacing-xl);
+        padding: var(--go-ui-spacing-lg);
+
+        .legend-item {
+            display: flex;
+            align-items: center;
+            gap: var(--go-ui-spacing-sm);
+
+            .icon {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                border-radius: 50%;
+                width: 1.4rem;
+                height: 1.4rem;
+                color: var(--go-ui-color-white);
+                font-size: 1.2rem;
```

Comment:
We should use CSS variable token

Reply:
Doesn't relate to any token

Reply:
All of the icons have 1.2 rem why not create a token for icon size